CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
 
 ### Jsonnet
 
+* [CHANGE] Alertmanager: Added a `PodDisruptionBudget` of `withMaxUnavailable = 1`, to ensure we maintain quorum during rollouts. #1683
```

Comment:
It's not a breaking change. I think we can switch this entry to `[ENHANCEMENT]`.